kubetest2-gke/deployer/build.go

```diff
@@ -39,9 +40,20 @@ var (
 	// $1 == prefix
 	// $2 == suffix
 	buildPrefix = regexp.MustCompile(`^(\d\.\d+\.\d+)([+-].*)?$`)
+
+	defaultImageTag = "gke.gcr.io"
```

Comment:
nit (can be done as followup), seems like `defaultImageRegistry` would be a better name